mne-tools/mne-connectivity
# [MRG] Bump website theme

closes #143

PR Description
--------------

- updates to use the latest patch release of the website theme
- remove some crufty templates
- switch to using JSON file for the version switcher dropdown (instead of `html_context`)

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [x] All CIs are happy
- [x] PR title starts with [MRG]
- [x] ~~[whats_new.rst](https://github.com/mne-tools/mne-connectivity/blob/main/doc/whats_new.rst) is updated~~ (not really necessary)
- [x] PR description includes phrase "closes <#issue-number>"
